website/landing/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,89 @@
+import { Box, Install, Resources, SandpackLogo, Text } from "../common";
+
+export const Hero: React.FC = () => {
+  return (
+    <Box
+      as="section"
+      css={{
+        alignItems: "center",
+        background: "$surface",
+        display: "flex",
+        flexDirection: "column",
+        gap: "48px",
+        height: "100%",
+        minHeight: "100vh",
+        paddingTop: "100px",
+        overflow: "hidden",
+      }}
+    >
+      <Box
+        css={{
+          alignItems: "center",
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "center",
+          gap: "40px",
+        }}
+      >
+        <Box css={{ width: "60px", "@bp1": { width: "100px" } }}>
+          <SandpackLogo theme="light" />
+        </Box>
+        <Box
+          css={{
+            alignItems: "center",
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            gap: "20px",
+          }}
+        >
+          <Text
+            as="h1"
+            css={{
+              fontWeight: "$semiBold",
+              fontSize: "36px",
+              lineHeight: "100%",
+              textAlign: "center",
+              letterSpacing: "-0.05em",
+
+              "@bp1": {
+                fontSize: "72px"
+              }
+            }}
+          >
+            Sandpack
+          </Text>
+          <Text
+            css={{
+              color: "$darkTextSecondary",
+              fontSize: "16px",
+              fontWeight: "$normal",
+              lineHeight: "19px",
+              letterSpacing: "-0.0125em",
+              textAlign: "center",
+              width: "280px",
+            }}
+          >
+            Bring the power of CodeSandbox live coding into your project.
```

Comment:
I'd bring all texts to a content file

Reply:
Addressed here: 686b67c8e3d7289935887594a386120bc62ca57c.